test/WebJobs.Script.Tests/Managment/FunctionsSyncManagerTests.cs

```diff
@@ -384,7 +384,20 @@ private static IFileSystem CreateFileSystem(ScriptApplicationHostOptions hostOpt
             fileSystem.SetupGet(f => f.File).Returns(fileBase.Object);
             fileBase.Setup(f => f.Exists(Path.Combine(rootPath, "host.json"))).Returns(true);
 
-            hostJsonContent = hostJsonContent ?? @"{ ""durableTask"": { ""HubName"": ""TestHubValue"", ""azureStorageConnectionStringName"": ""DurableStorage"" }}";
```

Comment:
Existing test is in place to verify SyncTriggers payload, but it was based on v1 host.json formatting. Updating this and the parsing keeps the test passing